packages/lesswrong/server/akismet.ts

```diff
@@ -88,7 +89,16 @@ getCollectionHooks("Comments").newAfter.add(async function checkCommentForSpamWi
     if (!currentUser) throw new Error("Submitted comment has no associated user");
     
     if (akismetKeySetting.get()) {
+      const start = Date.now();
+
       const spam = await checkForAkismetSpam({document: comment, type: "comment"})
+
+      const timeElapsed = Date.now() - start;
+      captureEvent('checkForAkismetSpam', {
```

Comment:
Vague convention is to have something more like `checkForAkismetSpamCompleted` since this happens after its done.